src/os/typedefs.h

```diff
@@ -30,7 +30,6 @@ typedef std::string  cstringT;
 
 typedef std::wstring stringT;
 typedef wchar_t charT;
-#define _S(x) L ## x
```

Comment:
Looks like the comment on 21-24 is now obsolete

Reply:
Yes, comment was removed.